migrations/20200322171627_tags.js

```diff
@@ -0,0 +1,53 @@
+exports.up = async knex => {
+  const buildTableSchema = [
+    {
+      tableName: "tag",
+      create: t => {
+        t.increments("id").primary();
+        t.text("group");
+        t.text("name").notNullable();
+        t.text("description").notNullable();
```

Comment:
is description 'friendly name' or is it a long detailed field?  If it's the display name, maybe call it either `title` or `display_name`?  description is fine, too, but then let's add some comments that describe what it's for (maybe in the models/tag.js file)

Reply:
I am not sure. The documentation named it as description. Open to modify it if needed.

Reply:
ok. let's leave it an wait until we have an interface.